tests/test_parser.py

```diff
@@ -17,6 +17,9 @@ class SingleParserTest(BaseTest):
     def test_number(self):
         self.assertEquals(self.parse("numfield=1"), Q(numfield=1))
 
+    def test_number_float(self):
+        self.assertEquals(self.parse("numfield=1.1"), Q(numfield=1.0))
```

Comment:
This test fails - I assume you were testing to make sure the test would fail, but then forgot to change it back to 1.0?